include/develop.php

```diff
@@ -1,6 +1,6 @@
+<p class="lead magick-description">The functionality of ImageMagick is typically utilized from the <a href="<?php echo $_SESSION['RelativePath']?>/../script/command-line-processing.php">command-line</a> or you can use the features from programs written in your favorite language. Choose from these interfaces: <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#ada">G2F</a> (Ada), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#c">MagickCore</a> (C), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#c">MagickWand</a> (C), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#ch">ChMagick</a> (Ch), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#com_">ImageMagickObject</a> (COM+), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#c__">Magick++</a> (C++), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#java">JMagick</a> (Java), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#javascript">WASM-ImageMagick</a> (Javascript/Typescript), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#julia">JuliaIO</a> (Julia), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#kmagick">KMagick</a> (Kotlin), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#lisp">L-Magick</a> (Lisp), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#lua">Lua</a> (LuaJIT), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#neko">NMagick</a> (Neko/haXe), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#dot-net">Magick.NET</a> (.NET), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#pascal">PascalMagick</a> (Pascal),  <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#perl">PerlMagick</a> (Perl), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#php">IMagick</a> (PHP), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#python">PythonMagick</a> (Python), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#r">magick</a> (R), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#ruby">RMagick</a> (Ruby), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#tcl">TclMagick</a> (Tcl/TK), or <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#wasm">WASM</a>. With a language interface, use ImageMagick to modify or create images dynamically and <var>automagically</var>.</p>
```

Comment:
This might be a good time to finally add some new lines here? Would you mind adding some after the `,`'s?

Reply:
Would a little extra formatting look better? E.g. like this?
```
The functionality of ImageMagick is typically utilized from the command-line or you can use the features from programs written in your favorite language.

Choose from these interfaces:
G2F (Ada),
MagickCore (C), 
KMagick (Kotlin),
... etc ...

With a language interface, use ImageMagick to modify or create images dynamically and automagically.
```

Reply:
Oh I did not mean to reformat it (no `</br>`) but just put all these interfaces on a new line to make it easier to see what is changed next time.

Reply:
Done, how's it look now?